src/System.Private.CoreLib/src/System/Threading/WaitHandle.cs

```diff
@@ -40,7 +36,7 @@ public abstract class WaitHandle : MarshalByRefObject, IDisposable
 
         private static IntPtr GetInvalidHandle()
         {
-            return Win32Native.INVALID_HANDLE_VALUE;
+            return Interop.Kernel32.INVALID_HANDLE_VALUE;
         }
         protected static readonly IntPtr InvalidHandle = GetInvalidHandle();
```

Comment:
This can be ` protected static readonly IntPtr InvalidHandle = Interop.Kernel32.INVALID_HANDLE_VALUE;` and `GetInvalidHandle()` can be deleted.

Reply:
Or `protected static readonly IntPtr InvalidHandle = new IntPtr(-1)` to make it more ready for sharing for CoreRT/Mono that do not have PAL/Kernel32 on Unix.

Reply:
Did the latter.